python/iceberg/api/expressions/residual_evaluator.py

```diff
@@ -115,3 +175,14 @@ def unbound_predicate(self, pred):
             return bound_residual
 
         return bound
+
+
+class UnpartitionedEvaluator(ResidualEvaluator):
+
+    def __init__(self, expr):
+        return super(UnpartitionedEvaluator, self).__init__(PartitionSpec.unpartitioned(),
```

Comment:
`return` from an `__init__`? And an expression after it? I guess you don't want the return statement here.

Reply:
yes, hmm...not sure what I was doing there, but you are of course right.  Will update.